app/api/api_v1/endpoints/shops.py

```diff
@@ -68,3 +68,17 @@ async def list_of_shops_for_users(
     if shops:
         return shops
     raise HTTPException(status_code=404, detail="Cannot find shops for your location.")
+
+
+@router.patch("/shops/{shop_id}", response_model=ShopDetails)
```

Comment:
Add `is_available` to `ShopDetails` pydantic model

Reply:
ok